src/lib/wikitools/Image.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  import { themeId } from '$lib/pickers/stores';
+
+  export let lightsrc: string;
+  export let darksrc: string;
+  export let alt: string;
+
+  let prefersDark: boolean = window.matchMedia(
+    '(prefers-color-scheme: dark)'
+  ).matches;
+</script>
+
+{#if $themeId === 'dark'}
+  <img src={darksrc} {alt} />
+{:else if $themeId === 'light'}
+  <img src={lightsrc} {alt} />
+{:else if $themeId === 'auto'}
+  {#if prefersDark}
```

Comment:
Sowas sollte eig. wegen DRY-Prinzip nicht existieren. Mach da lieber mal 'ne funktion irgendwo draus, die das für dich einfach nur theme: dark oder theme:light returned und dann muss man nicht in jedem component diesen "auto"-Tanz machen